source/common/tracing/zipkin/zipkin_core_types.h

```diff
@@ -452,6 +458,11 @@ class Span : public ZipkinBase {
    */
   bool debug() const { return debug_; }
 
+  /**
+   * @return whether or not the sampled attribute is set
+   */
+  bool sampled() const { return sampled_; }
```

Comment:
sampled is tri-state, so it can be absent (which means make the sampling decision here). If the code always ensures that, all good.